src/ga4gh/vrs/extras/annotator/vcf.py

```diff
@@ -179,8 +179,8 @@ def _update_vcf_header(
     @use_ga4gh_compute_identifier_when(VrsObjectIdentifierIs.MISSING)
     def annotate(
         self,
-        input_vcf_path: Path,
-        output_vcf_path: Path | None = None,
+        input_vcf_path: Path | str,
```

Comment:
can we constrain this to `Literal["-"]` or am I missing something

Reply:
Good thought